docker-compose/docker-compose.test.yml

```diff
@@ -27,17 +27,24 @@ services:
     ports:
       - "5000:80"
     depends_on:
-      - ms-sql-server
-      - rabbitmq
-      - seed-mssql
-    volumes:
-      - ../:/app
+      ms-sql-server:
+        condition: service_started
+      rabbitmq:
+        condition: service_started
+      seed-mssql:
+        condition: service_completed_successfully
+      azure-storage-emulator:
+        condition: service_started
+    configs:
+      - source: ConsumerApiConfig
+        target: app/appsettings.json
```

Comment:
Is this really necessary? Because this file should actually be copied into the output directory during the build process. If it doesn't there is something wrong

Reply:
![image](https://github.com/nmshd/backbone/assets/2041298/4d9b1fa3-490c-415a-915b-93c3b04edef0)
It seems the `appsettings.override.json` file is missing. How should I proceed?

Reply:
My comment was only for appsettings.json.